postgres/src/config.rs

```diff
@@ -307,6 +316,30 @@ impl Config {
         self.config.get_channel_binding()
     }
 
+    /// Sets the notice callback.
+    ///
+    /// This callback will be invoked with the contents of every
+    /// [`AsyncMessage::Notice`] that is received by the connection. Notices use
+    /// the same structure as errors, but they are not "errors" per-se.
+    ///
+    /// Notices are distinct from notifications, which are instead accessible
+    /// via the [`Notifications`] API.
+    ///
+    /// [`AsyncMessage::Notice`]: tokio_postgres::AsyncMessage::Notice
+    /// [`Notifications`]: crate::Notifications
+    pub fn notice_callback<F>(&mut self, f: F) -> &mut Config
+    where
+        F: Fn(DbError) + Clone + Send + 'static,
+    {
+        self.notice_callback = Box::new(f);
+        self
+    }
+
+    /// Gets the current notice callback.
+    pub fn get_notice_callback(&self) -> &(dyn FnMut(DbError) + Send) {
```

Comment:
This doesn't really seem to be that useful, right?

Reply:
Nope. Added just for consistency with the other accessors I added in #535. I don't particularly care about this, though, so happy to remove it if you prefer.

Reply:
Yeah, let's remove it for now then.

Reply:
Done.